### ISSUE TITLE
./pegleg lint - ignore file(s) functionality

### ISSUE THREAD

**Issue by darrendejaeger** [att-comdev/pegleg#1]
I've manually adjusted Pegleg on my local to ignore the ".DS_Store" file that is tossed into every Mac folder that the Finder application can see when I run ./pegleg lint. Perhaps it'd be worthwhile to have a blacklist of sorts? Maybe any file that matches .gitignore patterns?

**Comment by alanmeadows** [comment]
I agree Darren, there is already ignore code that can likely be poached from Armada for this purpose.
